### ISSUE TITLE
img2img, different results with same parameters, Diffusers vs A1111

### ISSUE THREAD

**Issue by zhndongyang** [huggingface/diffusers#6598]
### Describe the bug

In image to image feature, I found that different results with same parameters between Diffusers and A1111, and the results in Diffusers is not good. 

**Key Parameters**
Use model:  https://civitai.com/models/141122/raesanmix
Use Lora: https://civitai.com/models/110799/niji-19905

**The codes in Colab:** 
https://colab.research.google.com/drive/1MVEDYHD7-HA0EE7Var-43IZFKE6L9Oxl?usp=sharing

**Diffusers(Left) vs A1111(Right)**
<img width="1225" alt="diff" src="https://github.com/huggingface/diffusers/assets/6086729/459b955b-9467-4fb5-bfeb-d73c8751b9c3">
**Original Image:**
![4](https://github.com/huggingface/diffusers/assets/6086729/9fed2afd-5d1a-4a95-92fe-b4f23189240c)
**A1111 Settings**
![image](https://github.com/huggingface/diffusers/assets/6086729/17cde53d-7ef9-42af-a274-91ae8a843eb3)

### Reproduction

#build pipeline
import torch
from diffusers import EulerAncestralDiscreteScheduler, StableDiffusionImg2ImgPipeline

pipe = StableDiffusionImg2ImgPipeline.from_single_file(
    "./raesanmix_v41.safetensors", torch_dtype=torch.float16, use_safetensors=True)
pipe.load_lora_weights("./1990.safetensors", weight_name="1990.safetensors")
pipe.scheduler = EulerAncestralDiscreteScheduler.from_config(pipe.scheduler.config)
pipe = pipe.to("cuda")

#inference

from PIL import Image
from io import BytesIO
import requests

width=512
height=768
similarValue = 0.4
num_inference_steps=20
guidance_scale=7.5
generator = torch.Generator(device='cuda').manual_seed(0)
prompt = 'beautiful girl'

response = requests.get('https://huggingface.co/JoeyYang/resources/resolve/main/4.jpg?download=true')
init_image = Image.open(BytesIO(response.content)).convert("RGB")
init_image = init_image.resize((width, height))

images = pipe(prompt=prompt,
              # negative_prompt= negative_prompt,
              strength = similarValue, 
              image = init_image, 
              num_inference_steps=num_inference_steps,
              guidance_scale=guidance_scale,
              generator = generator,
              clip_skip = 2,
              ).images

images[0]

### System Info

- `diffusers` version: 0.25.0
- Platform: Linux-6.1.58+-x86_64-with-glibc2.35
- Python version: 3.10.12
- PyTorch version (GPU?): 2.1.0+cu121 (True)
- Huggingface_hub version: 0.20.2
- Transformers version: 4.30.2
- Accelerate version: 0.21.0
- xFormers version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

**Comment by sayakpaul** [comment]
Cc: @yiyixuxu

**Comment by yiyixuxu** [comment]
Hi:

Our noise construction is not the same as auto1111, so we don't expect a 1:1 matching output here. With one example it is hard for us to conclude whether it is worse or not. 

YiYi

**Comment by sayakpaul** [comment]
Is this still a problem?

**Comment by aiXia121** [comment]
> ### Describe the bug
> In image to image feature, I found that different results with same parameters between Diffusers and A1111, and the results in Diffusers is not good.
> 
> **Key Parameters** Use model: https://civitai.com/models/141122/raesanmix Use Lora: https://civitai.com/models/110799/niji-19905
> 
> **The codes in Colab:** https://colab.research.google.com/drive/1MVEDYHD7-HA0EE7Var-43IZFKE6L9Oxl?usp=sharing
> 
> **Diffusers(Left) vs A1111(Right)** <img alt="diff" width="1225" src="https://private-user-images.githubusercontent.com/6086729/297060567-459b955b-9467-4fb5-bfeb-d73c8751b9c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MTAwMDUsIm5iZiI6MTcwOTcwOTcwNSwicGF0aCI6Ii82MDg2NzI5LzI5NzA2MDU2Ny00NTliOTU1Yi05NDY3LTRmYjUtYmZlYi1kNzNjODc1MWI5YzMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDZUMDcyMTQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2FmODcyZGY5MmM1Njc5YjRkM2NmODVjNzA5ZjkwYTM1MmQ5YjEwNTVkNWFiNmU5MTA4MzFmOTg5MDg5ZTAwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.p8MXusaOxqvl-3YpuZN494nJgtPeoCFELrJvdawN1BQ"> **Original Image:** <img alt="image" width="598" src="https://private-user-images.githubusercontent.com/6086729/297062348-4842d1b3-278b-4354-88f9-a6d6fd558ac1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MTAwMDUsIm5iZiI6MTcwOTcwOTcwNSwicGF0aCI6Ii82MDg2NzI5LzI5NzA2MjM0OC00ODQyZDFiMy0yNzhiLTQzNTQtODhmOS1hNmQ2ZmQ1NThhYzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDZUMDcyMTQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDk3YTJmMTgxMWI4ZmY5NTJjZmIyZmU4ZDg0YmI1NTA3ZTZmZmM2ZDIxZmFkNGMxMzM2YjhmNDg5NGZjNzQ4YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.AjJLNv1qPmc8TFbKVfwL6fLFgpybzxw72uDw6WT-Tpc"> **A1111 Settings** ![image](https://private-user-images.githubusercontent.com/6086729/297060975-17cde53d-7ef9-42af-a274-91ae8a843eb3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MTAwMDUsIm5iZiI6MTcwOTcwOTcwNSwicGF0aCI6Ii82MDg2NzI5LzI5NzA2MDk3NS0xN2NkZTUzZC03ZWY5LTQyYWYtYTI3NC05MWFlOGE4NDNlYjMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDZUMDcyMTQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDUwMmUwYzNiZDBkYWVlMmViOGYwNjVkOTY3ZTg5MmM2YzM4ZDM1ZDM1YmI4NGFhOTU0ZmY2OTkzZTQ0ZWFiNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jSQnSyhQC_OLZzH8kQ9gp1PEV-nEwsFHQqjkEtE_xH0)
> 
> ### Reproduction
> ```
> #build pipeline
> import torch
> from diffusers import EulerAncestralDiscreteScheduler, StableDiffusionImg2ImgPipeline
> 
> 
> pipe = StableDiffusionImg2ImgPipeline.from_single_file(
>     "./raesanmix_v41.safetensors", torch_dtype=torch.float16, use_safetensors=True)
> pipe.load_lora_weights("./1990.safetensors", weight_name="1990.safetensors")
> pipe.scheduler = EulerAncestralDiscreteScheduler.from_config(pipe.scheduler.config)
> pipe = pipe.to("cuda")
> 
> 
> 
> 
> #inference
> 
> from PIL import Image
> from io import BytesIO
> import requests
> 
> width=512
> height=768
> similarValue = 0.4
> num_inference_steps=20
> guidance_scale=7.5
> generator = torch.Generator(device='cuda').manual_seed(0)
> prompt = 'beautiful girl'
> 
> response = requests.get('https://huggingface.co/JoeyYang/resources/resolve/main/4.jpg?download=true')
> init_image = Image.open(BytesIO(response.content)).convert("RGB")
> init_image = init_image.resize((width, height))
> 
> images = pipe(prompt=prompt,
>               # negative_prompt= negative_prompt,
>               strength = similarValue, 
>               image = init_image, 
>               num_inference_steps=num_inference_steps,
>               guidance_scale=guidance_scale,
>               generator = generator,
>               clip_skip = 2,
>               ).images
> 
> images[0]
> ```
> 
> ### Logs
> _No response_
> 
> ### System Info
> * `diffusers` version: 0.25.0
> * Platform: Linux-6.1.58+-x86_64-with-glibc2.35
> * Python version: 3.10.12
> * PyTorch version (GPU?): 2.1.0+cu121 (True)
> * Huggingface_hub version: 0.20.2
> * Transformers version: 4.30.2
> * Accelerate version: 0.21.0
> * xFormers version: not installed
> * Using GPU in script?:
> * Using distributed or parallel set-up in script?:
> 
> ### Who can help?
> _No response_


would mind show your pipe.scheduler or scheduler settings? or do you how to set the pipe's scheduler, they are some different.

@zhndongyang
